django/utils/log.py

```diff
@@ -65,17 +66,59 @@
 
 
 def configure_logging(logging_config, logging_settings):
+    def _replace_socket_handler(settings):
```

Comment:
Django should not be magically replacing handlers imo. We can document an alternative class but replacing a user choice is not the way to go.

Reply:
@candypoplatte what kind of data loss ?